server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/TcpClientInterface.js

```diff
@@ -29,6 +29,12 @@ class TcpClientInterface extends layerProtocol {
             remotePort;
             remoteProtocol;
 
+            static remoteProtocolEnum = {
+                HTTP: "tcp-client-interface-1-0:PROTOCOL_TYPE_HTTP",
+                HTTPS: "tcp-client-interface-1-0:PROTOCOL_TYPE_HTTPS",
+                NOT_YET_DEFINED: "tcp-client-interface-1-0:PROTOCOL_TYPE_NOT_YET_DEFINED"
+            };
```

Comment:
I think this should be part of OnfAttributes, at least to avoid duplicates (now it's in both, TcpClientInterface and TcpServerInterface).

Reply:
As , the prefix onf notation of the values are diffrent for tcp-server and tcp-client declared them respectively in TcpServerInterface and TcpClientInterface